.CI/common.groovy

```diff
@@ -43,7 +43,7 @@ def numLogicalCPU() {
 }
 
 void partest(cache=true, extraArgs='') {
-  sh "rm -f omc-diff.skip && make -C testsuite/difftool clean && make --output-sync -C testsuite/difftool"
+  sh "rm -f omc-diff.skip && ${makeCommand()} -C testsuite/difftool clean && make --output-sync -C testsuite/difftool"
```

Comment:
What about the second call to make in this line?

Reply:
I don't know how I could have missed that :D